docs/tools/vdb_table/data/nucliadb.json

```diff
@@ -9,7 +9,7 @@
     "slug": "nucliadb"
   },
   "oss": {
-    "support": "",
+    "support": "full",
```

Comment:
please provide the github repo link in the source_url here

Reply:
Repo link added